lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -91,6 +91,8 @@ private void Run()
                 .AddDriverConfiguration(driverConfig.Build())
                 .AddGlobalAssemblyForType(typeof(HelloDriverYarn))
                 .SetJobIdentifier("HelloREEF")
+                .AddJobSubmissionEnvVariable(Environment.PATH.ToString(), "value1")
```

Comment:
What are these changes in the example for?

Reply:
It is to show people example and also test the code path.